### DISCUSSION TITLE
news/20231212/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#304] [Website Discussions]
# news/20231212/

CoreMelt has released CoreMelt Manager v1.1.4.

https://fcp.cafe/news/20231212/

**Comment by simplevideoedit** [comment]
Transcriber is one of the best purchases I made for FCP. It enabled me to stop using REV for transcribing so it has already saved me $ and time. Always excited to see what else Alex is brewing. Thank goodness for all the smart people keeping FCP alive!

**Comment by shylock74** [comment]
Look, receiving feedback messages like this is something that really fills my heart.
Just being called a "genius" in an article almost embarrasses me. I spend my life trying to stay under the radar.... Yes, of course, in some ways I am also a "public" figure, although my audience is really limited. Despite being almost 50 years old, I have a hard time realizing that some of the things I do are being used by other people and, and it is a miracle every time it happens, the tools I have made can serve to improve if not the lives at least the work of others... Or someone else's...
Therefore, coming into contact with others who appreciate what I do not only fills me with joy but, to a great extent, destabilizes me.... Have mercy on me, I am not used to being treated so well!
Given that, I thank you again for this feedback. I am constantly working to improve this application more and more, making it more performant, feature-rich and worthwhile.
A very strong hug, and since we are in the home stretch, a strong wish for the New Year and, trying to be as politically correct as possible, a neutral "happy holidays"!

Alex

**Reply by latenitefilms** [reply]
Onwards & Upwards!
